packages/components/app/styles/components/tooltip.scss

```
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// TOOLTIP & TOOLTIP-BUTTON
+//
+
+@use "../mixins/focus-ring" as *;
+
+:root {
+  // Dimensions:
+  --token-tooltip-max-width: 280px;
+
+  // Focus highlight position
+  --token-tooltip-tab-focus-offset: -2px;
+
+  // Colors:
+  --token-tooltip-surface-primary: var(--token-color-palette-neutral-700);
```

let's use the same naming convention used for the `SideNav` ([reference](https://github.com/hashicorp/design-system/blob/main/packages/tokens/dist/products/css/tokens.css#L267-L272)):
```suggestion
  --token-side-nav-color-foreground-primary: var(--token-color-foreground-high-contrast);
  --token-tooltip-color-surface-primary: var(--token-color-palette-neutral-700);
```

---

Shouldn't it be "background" instead of "foreground"?

---

No. `foreground` is for the text; `surface` is for the background. See other color tokens: 
https://helios.hashicorp.design/foundations/colors#using-semantic-colors

---

**Update:** I pushed an update to the name but used "background" for now.

---

The tooltip uses a dark background with light text.